src/openlcb/SimpleStack.cxx

```diff
@@ -116,6 +116,7 @@ void SimpleStackBase::start_stack(bool delay_start)
     if (!delay_start)
     {
         start_iface(false);
+        node()->initialize();
```

Comment:
i would expect to iterate the iface()->local_nodes list and call initialize on all of them. This would ensure that all virtual nodes get initialized as well.